x-pack/distributions/internal/observabilitySRE/qa/smoke/docker/docker-compose.yml

```diff
@@ -18,6 +18,8 @@ services:
       - xpack.security.transport.ssl.certificate_authorities=/usr/share/elasticsearch/config/certs/ca.crt
     ports:
       - "9200:9200"
+    entrypoint: ["/usr/local/bin/docker-entrypoint.sh"]
```

Comment:
I proposed a change upstream to avoid this https://github.com/elastic/elasticsearch/pull/127788